transports/p2p/server.go

```diff
@@ -325,17 +325,17 @@ func (sp *serverPeer) OnVersion(_ *peer.Peer, msg *wire.MsgVersion) *wire.MsgRej
 		return wire.NewMsgReject(msg.Command(), wire.RejectNonstandard, reason)
 	}
 
-	// Reject outbound peers that are not full nodes.
-	wantServices := wire.SFNodeNetwork
-	if !isInbound && !hasServices(msg.Services, wantServices) {
-		missingServices := wantServices & ^msg.Services
-		sp.server.p2pConfig.Logger.Debugf("Rejecting peer %s with services %v due to not "+
-			"providing desired services %v", sp.Peer, msg.Services,
-			missingServices)
-		reason := fmt.Sprintf("required services %#x not offered",
-			uint64(missingServices))
-		return wire.NewMsgReject(msg.Command(), wire.RejectNonstandard, reason)
-	}
+	// This would allow rejection of non full node peers, but we don't want any of the services so no reason to reject outbound peers.
+	// wantServices := wire.SFNodeNetwork
```

Comment:
note: we do not need to connect to full nodes to get headers. This should allow Pulse instances to connect with one another and (maybe) even send eachother header messages.

Reply:
If we don't need it then I would suggest to remove this code entirely instead of commenting it out.